spa/src/components/modals/PersonalInfoModal.vue

```diff
@@ -50,6 +50,10 @@
               </select>
               </td>
             </tr>
+            <tr>
+              <td><b>Default Chat</b></td>
+              <td><b>2D</b> <input type="radio" v-model="info.chatdefault" value="0">&nbsp;<b>3D</b> <input type="radio" v-model="info.chatdefault" value="1"></td>
+            </tr>
```

Comment:
Can we add a note that the user needs to refresh their browser for a change in default chat to take effect?